DependencyInjection/Configuration/Configuration.php

```diff
@@ -267,7 +268,7 @@ private function addProfilerStorageSection(ArrayNodeDefinition $rootNode)
         $rootNode
             ->children()
                 ->arrayNode('profiler_storage')
-                    ->setDeprecated('Redis profiler storage is not available anymore since Symfony 4.4')
+                    ->setDeprecated(...$this->getProfilerStorageDeprecationMessage())
```

Comment:
This isn't going to work on PHP < 7.4 right?

Reply:
Sorry, I don't get it, you mean [argument unpacking](https://www.php.net/manual/en/functions.arguments.php#functions.variable-arg-list) via `...`? It will work since PHP 5.6 - https://3v4l.org/XSsm4

Reply:
hu ho, sorry I wasn't aware of that.
I was thinking of https://wiki.php.net/rfc/spread_operator_for_array